sdk/script/management-sdk.ts

```diff
@@ -130,6 +130,11 @@ class AccountManager {
             .then(() => null);
     }
 
+    public removeSessions(createdBy: string): Promise<void> {
```

Comment:
Should we call this `removeSession`? I appreciate that on the server there could be multiple sessions for the same machine, but the CLI gives the impression that there is a 1:1 relationship between machine names and sessions, so if feels like the SDK should do the same thing.